core/validation/pre_transformation_validate.py

```diff
@@ -1,3 +1,10 @@
+"""Module for methods relating to pre-transformation validation.
+
+Pre-transformation validation occurs before any stages in the ETL pipeline.
+It validates user input based on the data as it appears in a dictionary of
+DataFrames read directly from the submission file.
+"""
```

Comment:
May be able to get across a bit clearer that it's primary purpose is to do some high level "submission-level" checks before extraction/transformation and validation of the data itself